pkgs/applications/version-management/pijul/pijul.org.context

```diff
@@ -0,0 +1,92 @@
+
+Context:
```

Comment:
`darcs changes --context` will give you this. This is needed because we're not building from a tag so darcs needs to know how to apply the patches to get to the commit we want.